client/src/domain/event/EventLocation.tsx

```diff
@@ -0,0 +1,156 @@
+      ","
+    )}`.replace(/\s/g, "+");
+  };
+
+  const getGoogleDirectionsLink = () => {
+    const location = eventData.linkedEventsEventDetails.location;
+    const streetAddress = getLocalisedString(
+      (location && location.streetAddress) || {},
+      locale
+    );
+    const postalCode = (location && location.postalCode) || {};
+    const addressLocality = getLocalisedString(
+      (location && location.addressLocality) || {},
+      locale
+    );
+    const coordinates =
+      location && location.position
+        ? // Get coordinates for Google in correct order
+          [...location.position.coordinates].reverse()
+        : [];
+
+    return `https://www.google.com/maps/dir//${streetAddress},+${postalCode}+${addressLocality}/@${coordinates.join(
+      ","
+    )}`.replace(/\s/g, "+");
+  };
+
+  const getHslDirectionsLink = () => {
+    const location = eventData.linkedEventsEventDetails.location;
+    const streetAddress = getLocalisedString(
+      (location && location.streetAddress) || {},
+      locale
+    );
+    const addressLocality = getLocalisedString(
+      (location && location.addressLocality) || {},
+      locale
+    );
+    const coordinates =
+      location && location.position
+        ? // Get coordinates for HSL in correct order
+          [...location.position.coordinates].reverse()
+        : [];
+
+    return `https://reittiopas.hsl.fi/%20/${encodeURIComponent(
+      streetAddress
+    )},%20${encodeURIComponent(addressLocality)}::${coordinates.join(
+      ","
+    )}?locale=${locale}`;
+  };
+
+  const getLocationStr = () => {
+    const location = eventData.linkedEventsEventDetails.location;
+    const addressLocality = getLocalisedString(
+      (location && location.addressLocality) || {},
+      locale
+    );
+    const district =
+      location &&
+      location.divisions.find(
+        division => ["district", "neighborhood"].indexOf(division.type) !== -1
+      );
+    const districtStr =
+      district && district.name
+        ? getLocalisedString(district.name, locale)
+        : null;
+    const streetAddress = getLocalisedString(
+      (location && location.streetAddress) || {},
+      locale
+    );
+
+    return [streetAddress, districtStr, addressLocality]
+      .filter(e => e)
+      .join(", ");
+  };
+
+  const coordinates =
+    eventData.linkedEventsEventDetails.location &&
+    eventData.linkedEventsEventDetails.location.position
+      ? eventData.linkedEventsEventDetails.location.position.coordinates
+      : null;
+  const name = eventData.linkedEventsEventDetails.name;
+
+  return (
+    <div className={styles.eventLocationContainer}>
+      <div className={styles.titleWrapper}>
+        <div className={styles.title}>
+          <LocationIcon />
+          <h3>{t("event.location.title")}</h3>
+        </div>
+        <a className={styles.mapLink} href={getGoogleLink()} target="__blank">
```

Comment:
`__blank` should be just `_blank` with a single underscore. Remember to add `rel="noopener noreferrer"`